licenses/LICENSE-BIN

```diff
@@ -0,0 +1,281 @@
+and any and all patent rights owned or freely licensable by each licensor hereunder covering either
+(i) the unmodified Software as contributed to or provided by such licensor, or
+(ii) the Larger Works (as defined below), to deal in both
+(a) the Software, and
+(b) any piece of software and/or hardware listed in the lrgrwrks.txt file if one is
+included with the Software each a "Larger Work" to which the Software is contributed by such licensors),
+without restriction, including without limitation the rights to copy,
+create derivative works of, display, perform, and distribute the Software
+and make, use, sell, offer for sale, import, export, have made, and have sold the Software and the Larger Work(s),
+and to sublicense the foregoing rights on either these or other terms.
+
+This license is subject to the following condition:
+
+The above copyright notice and either this complete permission notice or at a minimum a reference to the UPL
+must be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+------
+This product has a bundle spdlog, which is available under the MIT License.
+The source code of spdlog can be found at https://github.com/gabime/spdlog.
+
+The MIT License (MIT)
+
+Copyright (c) 2016 Gabi Melman.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+------
+This product has a bundle substratevm, which is available under the GPL License.
+The source code of spdlog can be found at https://github.com/oracle/graal/tree/master/substratevm.
```

Comment:
GPL is not compatible with ASL2, right?

Reply:
WHAT CAN WE NOT INCLUDE IN AN ASF PROJECT ? please see https://apache.org/legal/resolved.html

Reply:
@justinmclean Could you please help to check about this issue? What should I do to 

> GPL is not compatible with ASL2, right?

the substratevm is under GPL 2 with Classpath exception, i am not sure that if it can used here.  https://github.com/oracle/graal/blob/master/substratevm/LICENSE.md

Reply:
> WHAT CAN WE NOT INCLUDE IN AN ASF PROJECT ? please see https://apache.org/legal/resolved.html

In this page, CATEGORY X : GNU GPL 1,2,3, , i found "Special exceptions to the GNU GPL (e.g. GNU Classpath) unless otherwise permitted elsewhere on this page.", I am not sure it is Ok.